tasks/install-alert_manager.yml

```diff
@@ -1,5 +1,18 @@
 - include_vars: 'alertmanager.yml'
 
+- name: Creating alert manager directories
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: "{{ prometheus_owner }}"
+    group: "{{ prometheus_group }}"
+    mode: 0750
+  with_items:
+    - "{{ prometheus_config_dir }}"
+    - "{{ prometheus_alert_manager_data_dir }}"
```

Comment:
If you are taking non recursive approach then 

prometheus_alert_manager_data_dir: "{{ prometheus_lib_dir }}/alertmanager" requires `prometheus_lib_dir` as well